settings.gradle

```diff
@@ -34,6 +34,8 @@ include "async"
 include "batch"
 include "cloud-function-web"
 include "cloud-function-webflux"
+include "cloud-stream-kafka"
```

Comment:
Such entries must be added to `ci/smoke-tests.yaml`, too.

Reply:
Yeh I was about to add it, as I did see the file, but was [unsure](https://github.com/spring-projects/spring-aot-smoke-tests/pull/79#issue-1331426712).